scripts/exregional_save_restart.sh

```diff
@@ -112,28 +112,31 @@ restart_prefix=${save_yyyy}${save_mm}${save_dd}.${save_hh}0000
 if [ ! -r ${nwges_dir}/INPUT/gfs_ctrl.nc ]; then
   cp_vrfy $run_dir/INPUT/gfs_ctrl.nc ${nwges_dir}/INPUT/gfs_ctrl.nc
   if [ -r ${run_dir}/INPUT/coupler.res ]; then  # warm start
-    if [ "${IO_LAYOUT_Y}" == "1" ]; then
-      for file in ${filelistn}; do
-        cp_vrfy $run_dir/INPUT/${file} ${nwges_dir}/INPUT/${file}
-      done
-    else
-      for file in ${filelistn}; do
-        for ii in ${list_iolayout}
-        do
-          iii=$(printf %4.4i $ii)
-         cp_vrfy $run_dir/INPUT/${file}.${iii} ${nwges_dir}/INPUT/${file}.${iii}
+    if [ "${DO_SAVE_INPUT}" = TRUE ]; then
+      if [ "${IO_LAYOUT_Y}" == "1" ]; then
+        for file in ${filelistn}; do
+          cp_vrfy $run_dir/INPUT/${file} ${nwges_dir}/INPUT/${file}
+        done
+      else
+        for file in ${filelistn}; do
+          for ii in ${list_iolayout}
+          do
+            iii=$(printf %4.4i $ii)
+           cp_vrfy $run_dir/INPUT/${file}.${iii} ${nwges_dir}/INPUT/${file}.${iii}
+          done
         done
+      fi
+      for file in ${filelist}; do
+        cp_vrfy $run_dir/INPUT/${file} ${nwges_dir}/INPUT/${file}
       done
     fi
-    for file in ${filelist}; do
-      cp_vrfy $run_dir/INPUT/${file} ${nwges_dir}/INPUT/${file}
-    done
   else  # cold start
     for file in ${filelistcold}; do
```

Comment:
@chunhuazhou @hu5970  I think we also need to skip saving cold start INPUT files, right?

Reply:
Yes

Reply:
Just changed exregional_save_restart.sh to include saving cold start INPUT. Please take a look and let me know if you have further comments!